lightningd/anchorspend.c

```diff
@@ -204,7 +204,12 @@ struct anchor_details *create_anchor_details(const tal_t *ctx,
 			continue;
 
 		v.msat = hout->msat;
-		v.block = hout->cltv_expiry;
+		/* Our real deadline here is the INCOMING htlc.  If it's us, use the default so we don't leak
+		 * too much information about it. */
```

Comment:
I didn't quite follow the comment. This is to set the fees as though we could be a forwarding node even if we initiated the payment?

Reply:
There's actually no real deadline if we're the source, since there's no incoming HTLC.  But we use the default (12 blocks) which is actually what we use for most forwards anyway: if we didn't set some deadline it would become pretty clear we didn't care.